lib/Java2ClientGenerator.php

```diff
@@ -1209,6 +1217,9 @@ public function getInitialPropertyValue($propertyNode)
 		case "time":
 			return "Long.MIN_VALUE";
 		case "int":
+			$isTime = $propertyNode->hasAttribute("isTime");
```

Comment:
do this handeling outside the case and assign time propType instead
worth even checking where prop type is originally set and change to time there on one place and not multiple